browser-es6.js

```diff
@@ -21,5 +21,12 @@ module.exports = merge(base, {
   },
   plugins: [
     'react'
-  ]
+  ],
+  rules: {
+    'new-cap': 0,
```

Comment:
After reading the docs, I agree with you here, I'll simply add specific vars via `capIsNewExceptions` instead of using `0` here.